LiveSplit.AutoSplitters.xml

```diff
@@ -14144,4 +14144,14 @@
         <Website>https://github.com/ZywelZill/zywel-zill-autosplitters</Website>
         <Description>AutoSplitter By Zywel Zill)</Description>
     </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>Eye of the Temple</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/runevision/EyeOfTheTempleAutoSplitter/main/EyeOfTheTemple.asl</URL>
```

Comment:
This will not work! This auto splitter requires the presence of [`just-ero/AutoSplitHelp/libraries/ULibrary.bin`](https://github.com/just-ero/AutoSplitHelp/blob/main/libraries/ULibrary.bin) in the user's files. To ensure this, please add this tag BELOW your raw ASL link;

```xml
<URL>https://raw.githubusercontent.com/just-ero/AutoSplitHelp/main/libraries/ULibrary.bin</URL>
```

Reply:
Should I make a new PR or is there a way to add this change to this PR even though it was already merged?

Reply:
You should make a new PR.

Reply:
Done:
https://github.com/LiveSplit/LiveSplit.AutoSplitters/pull/689